source/iterator/constants.go

```diff
@@ -42,34 +42,47 @@ const (
 	querySelectRowsFmt     = "SELECT %s FROM %s%s ORDER BY %s ASC FETCH NEXT %d ROWS ONLY"
 	queryDeleteByIDs       = "DELETE FROM %s WHERE %s IN (%s)"
 
-	queryTrackingTableExtendWithConduitColumns = `ALTER TABLE %s 
-ADD (
-	%s NUMBER GENERATED BY DEFAULT ON NULL AS IDENTITY,
-    %s VARCHAR2(6) NOT NULL CHECK(%s IN ('insert','update','delete')),
-    %s DATE DEFAULT SYSDATE,
-	
-	CONSTRAINT %s UNIQUE (%s)
-)`
+	queryTrackingTableExtendWithConduitColumns = `
```

Comment:
A note about this file generally: it looks like most constants here are used in only one place (snapshot or CDC), so maybe it makes sense just having them there.

Reply:
Yep, it's better.
Done.